ql.mm

```diff
@@ -10393,25 +9869,25 @@ Kalmbach axioms (soundness proofs) that require WOML
     TWDVFWDVTAUTVALOSNWEPVOWEPWDVFPWCVAPWCPAUGUHHHUINWFVOVFVBEZVGEZVGVOULVMWGVG
     VBVFMSWHVFVBVGEZEZVGVFVBVGLWJVFUTVABEZEZEVFVGEVGWIWLVFWLWIUTVABUJOQWLVGVFWK
     BUTVABVQRQQVFVGVRRTUMTTTUNVKVBVLVIABUOABUPUQABURUS $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 2, and j is set to 0.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 2, and j is set to 0.  (Contributed by Roy F. Longton, 1-Jul-2005.)
```

Comment:
The Contributed line should be 3-Jul-2005 and there should be no Revised by. I realize that the original has both dates but there's no point in having two dates two days apart.